src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -22,6 +24,14 @@ export class HmctsGlobalHeaderComponent {
         ]
     };
 
-    constructor() { }
+    constructor(public store: Store<fromRoot.State> ) { }
+
+  onNavigate(path) {
+      this.store.dispatch(new fromRoot.Go(
+        {
+          path: ['/register', path]
+        }
+      ));
+  }
```

Comment:
i told u wrong this is only used when we dispatching navigation from effects

Reply:
removed redundant code